CRM/Event/BAO/Participant.php

```diff
@@ -315,8 +315,7 @@ public static function create(&$params) {
    *                 calculation or not. (it is for cron job  purpose)
    *
    * @param bool $returnWaitingCount
-   * @param bool $considerTestParticipant
-   *   When TRUE, include participant records where is_test = 1.
+   * @param bool $considerTestParticipant deprecated, unused
```

Comment:
this is FALSE in all core usages https://lab.civicrm.org/dev/core/-/issues/4907 - it is hard to imagine a non-core usage of this internal core function that warrants us keeping this parameter active / including it in our final cut